scintellometry/io/arochime.py

```diff
@@ -35,9 +39,20 @@ def __init__(self, raw_files, blocksize,
         self.fedge = fedge
         self.fedge_at_top = fedge_at_top
         f = fftshift(fftfreq(nchan, (2./samplerate).to(u.s).value)) * u.Hz
-        if fedge_at_top:
+        if fedge_at_top :
             self.frequencies = fedge - (f-f[0])
-        else:
+            if self.time0.value > 1406764800  and self.time0.value < 1406937600:
```

Comment:
It is a bit ugly, but how about adding to the if statement that `aro` should be in the file name? I.e.,
```
if 'aro' in raw_files[0] and (Time ... < self.time0 < Time ...):
```